### PR TITLE
[AAE-2549] Incorrect response when completing task using the same correlation key for Intermediate Message Event

### PR DESCRIPTION
**Pull request by AkhileshPamidimarthi**
Earlier when same value was being passed in the text field of the form. We were getting status code as 500(Internal Server Error).
This fix resolved this issue and now generating the status code as 409(Conflict).

https://github.com/Activiti/Activiti/issues/4347

### CODE REVIEW

**Comment by miguelruizdev** [comment]
Please, provide a description of the changes and commit according to this guide: https://alfresco.atlassian.net/wiki/spaces/HXP/pages/1281163265/How+to+commit+and+create+a+PR+in+Activiti+GitHub+Organization

File: activiti-cloud-acceptance-scenarios/runtime-acceptance-tests/src/main/java/org/activiti/cloud/qa/story/ProcessInstanceMessages.java [hunk 1]
```diff
@@ -15,8 +15,7 @@
  */
 package org.activiti.cloud.qa.story;
 
-import static org.activiti.cloud.acc.core.assertions.RestErrorAssert.assertThatRestInternalServerErrorIsThrownBy;
-import static org.activiti.cloud.acc.core.assertions.RestErrorAssert.assertThatRestNotFoundErrorIsThrownBy;
+import static org.activiti.cloud.acc.core.assertions.RestErrorAssert.*;
```
Comment (matteo4diani) [inline activiti-cloud-acceptance-scenarios/runtime-acceptance-tests/src/main/java/org/activiti/cloud/qa/story/ProcessInstanceMessages.java]: Revert all the wildcard imports and check your IDE settings, when optimising imports IntelliJ must default to single class import -> https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

File: activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/RuntimeBundleExceptionHandler.java [hunk 2]
```diff
@@ -15,12 +15,7 @@
  */
 package org.activiti.cloud.services.rest.controllers;
 
-import static org.springframework.http.HttpStatus.BAD_REQUEST;
-import static org.springframework.http.HttpStatus.FORBIDDEN;
-import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
-import static org.springframework.http.HttpStatus.NOT_FOUND;
-import static org.springframework.http.HttpStatus.NO_CONTENT;
-import static org.springframework.http.HttpStatus.UNPROCESSABLE_ENTITY;
+import static org.springframework.http.HttpStatus.*;
```
Comment (matteo4diani) [inline activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/RuntimeBundleExceptionHandler.java]: Revert this to single-class imports

**Review by matteo4diani — CHANGES_REQUESTED** [review]

**Review by matteo4diani — APPROVED** [review]
LGTM 👍🏼 Good job @AkhileshPamidimarthi

**Review by miguelruizdev — APPROVED** [review]
LGTM! Congrats on your first PR!

**Pull request merged**